src/crystal/system/process.cr

```diff
@@ -0,0 +1,9 @@
+# :nodoc:
+struct Crystal::System::Process
```

Comment:
Can you please add an API description with commented out method signatures like in similar files?

Reply:
OK, take a look.